index.js

```diff
@@ -158,8 +158,9 @@ NYC.prototype._prepGlobPatterns = function (patterns) {
 
 NYC.prototype.addFile = function (filename) {
   var relFile = path.relative(this.cwd, filename)
-  var source = stripBom(fs.readFileSync(filename, 'utf8'))
+  var source = this._readTranspiledSource(path.resolve(this.cwd, filename))
```

Comment:
My only thought would be to do this conditionally if we don't already have coverage data on that particular file. This feels like a performance hit.